### PR TITLE
Milestone 2 : Add middle and contact section

### PR DESCRIPTION
**Pull request by Chrissiku**
For this part of the project, I implement the middle section of my portfolio mobile for the mobile section and I practice more on the flex-box and Grid layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,17 +71,165 @@ <h3 class="title">Skills</h3>
...
+          </li>
+        </ul>
         <div class="work">
           <div><i class="fa-solid fa-arrow-down"></i><span> Work</span></div>
         </div>
       </div>
     </section>
+
+    <!-- Middle part -->
+    <section class="middle-section">
```
Comment (OLIPLICHE) [inline index.html:98]: - [ ] Pease The main container of the projects has a display: grid

File: index.html [hunk 2]
```diff
@@ -71,17 +71,165 @@ <h3 class="title">Skills</h3>
...
+              <ul>
+                <li><a href="">css</a></li>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+              </ul>
+            </div>
+            <button type="button" class="btn">
+              <span>See Project</span>
+            </button>
```
Comment (OLIPLICHE) [inline index.html:126]: - [ ] Please The button  "See project"  follows the button styling: Active, Hover, and Pressed 
- [ ] see design: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)


kk

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Chrissiku, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request well defined
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Chrissiku 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**